packages/backend/src/asset/service.test.ts

```diff
@@ -138,36 +142,49 @@ describe('Asset Service', (): void => {
 
   describe('update', (): void => {
     describe.each`
-      withdrawalThreshold
-      ${null}
-      ${BigInt(0)}
-      ${BigInt(5)}
+      withdrawalThreshold | liquidityThreshold
+      ${null}             | ${null}
+      ${BigInt(0)}        | ${null}
+      ${BigInt(5)}        | ${null}
+      ${null}             | ${BigInt(0)}
+      ${null}             | ${BigInt(5)}
+      ${BigInt(0)}        | ${BigInt(0)}
+      ${BigInt(5)}        | ${BigInt(5)}
     `(
-      "Asset's withdrawal threshold can be updated from $withdrawalThreshold",
-      ({ withdrawalThreshold }): void => {
+      'Asset threshold can be updated from withdrawalThreshold: $withdrawalThreshold, liquidityThreshold: $liquidityThreshold',
+      ({ withdrawalThreshold, liquidityThreshold }): void => {
         let assetId: string
 
         beforeEach(async (): Promise<void> => {
           const asset = await assetService.create({
             ...randomAsset(),
-            withdrawalThreshold
+            withdrawalThreshold,
+            liquidityThreshold
           })
           assert.ok(!isAssetError(asset))
           expect(asset.withdrawalThreshold).toEqual(withdrawalThreshold)
```

Comment:
Should the test also check that the value of `liquidityThreshold` is correct along with `withdrawalThreshold`?